recipe/meta.yaml

```diff
@@ -44,6 +44,8 @@ test:
   # https://github.com/conda-forge/staged-recipes/pull/292#issuecomment-208080893.
   commands:                # [not win]
     - nosetests zmq.tests  # [not win]
+  imports:                 # [win]
+    - zmq                  # [win]
```

Comment:
FWIW I think the other changes proposed here look reasonable. If there is something off about the Windows Python 3.5 build, then we should drop the special casing of it. I'm just not sure the best way to test it.

It could be that there is something that changes about `zeromq` that doesn't work. Let's take a look at that.